Userland/Libraries/LibWeb/WebDriver/Error.cpp

```diff
@@ -33,6 +33,7 @@ static Vector<ErrorCodeData> const s_error_code_data = {
     { ErrorCode::NoSuchFrame, 404, "no such frame" },
     { ErrorCode::NoSuchWindow, 404, "no such window" },
     { ErrorCode::NoSuchShadowRoot, 404, "no such shadow root" },
+    { ErrorCode::NotImplemented, 501, "not implemented" },
```

Comment:
this isn't needed, `UnsupportedOperation` exists

Reply:
Okay, I thought there was a difference between "Unsupported" and "Not Implemented", maybe meaning that unsupported just means the engine doesn't support it by design/* and not implemented is more of a temp thing? What do you think? I have no problem in removing this if you think its unnecessary :^)

Reply:
> Indicates that a [command](https://w3c.github.io/webdriver/#dfn-commands) that should have executed properly cannot be supported for some reason.

fits perfectly, there's no reason to invent our own error code for this